code/modules/lighting/lighting_object.dm

```diff
@@ -10,6 +10,7 @@
 	mouse_opacity = MOUSE_OPACITY_TRANSPARENT
 	layer            = LIGHTING_LAYER
 	invisibility     = INVISIBILITY_LIGHTING
+	vis_flags = VIS_INHERIT_LAYER
```

Comment:
:eyes: what's this

Reply:
i mentioned it in the OP too but this fixed the issue I had, it didn't /seem/ to affect multiz in my A/B tests

Reply:
i mentioned it in the OP too but this fixed the issue I had, it didn't /seem/ to affect multiz in my A/B tests

Reply:
I don't like changing things without knowing why they work though :sadge:

Reply:
okay so this is moving the lighting layer of the vis contents objects to the same plane as the portal, which means it renders under the main lighting layer.

So you get bleed through but it doesn't double add.

Reply:
VIS_HIDE is also possible, my main concern is messing up multiz

Reply:
also did you try `VIS_INHERIT_ID`

Reply:
i'll give it a go